custom/icds_reports/models/aggregate.py

```diff
@@ -415,6 +415,8 @@ class AggAwc(models.Model, AggregateMixin):
     infra_type_of_building_kuccha = models.IntegerField(null=True)
     infra_type_of_building_partial_covered_space = models.IntegerField(null=True)
     infra_clean_water = models.IntegerField(null=True)
+    toilet_facility = models.IntegerField(null=True)
```

Comment:
This is a `smallint` in your SQL migration

Reply:
Good catch!.  I would make them all ints as small int range is Less.